integration-tests/authorizenet_test.go

```diff
@@ -173,4 +174,47 @@ func TestAuthNetAuthVoid(t *testing.T) {
 	}
 }
 
-// TestAuthNetAuthCaptureRefund cannot be written since there is no way to hack settlement and funds cannot refund until they settle
+// TestAuthNetAuthCaptureRefund
+func TestAuthNetAuthCaptureRefund(t *testing.T) {
+	client := authorizenet.NewClient(getEnv("AUTH_NET_LOGIN_ID"), getEnv("AUTH_NET_TXN_KEY"), common.Sandbox)
+	authRequest := sleet_testing.BaseAuthorizationRequestWithEmailPhoneNumber()
+	authRequest.Amount.Amount = int64(randomdata.Number(100))
+	auth, err := client.Authorize(authRequest)
+	if err != nil {
+		t.Error("Authorize request should not have failed")
+	}
+	if !auth.Success {
+		t.Error("Resulting auth should have been successful")
+	}
+
+	captureRequest := &sleet.CaptureRequest{
+		Amount:               &authRequest.Amount,
+		TransactionReference: auth.TransactionReference,
+	}
+	capture, err := client.Capture(captureRequest)
+	if err != nil {
+		t.Error("Capture request should not have failed")
+	}
+	if !capture.Success {
+		t.Error("Resulting capture should have been successful")
+	}
+
+	// Refunds for AuthNet take 24 hours to settle. The only option for immediate testing is to do a non-transaction
+	// referenced refund.
+	refundRequest := &sleet.RefundRequest{
+		Amount:               &authRequest.Amount,
+		Last4: authRequest.CreditCard.Number,
```

Comment:
Are we just overloading last4 with the full number for this test? If so, can we call this out in the comment on line 202?